### PR TITLE
docs: add GDE profile of Fabian Gosebrink

### PR DESCRIPTION
**Pull request by FabianGosebrink**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [x] Other... Please describe:

Added GDE Profile of Fabian Gosebrink 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by Splaktar — COMMENTED** [review]
I think that just `docs: ...` is good for the commit message. I don't think that `docs-infra` should be used.

You can just reword the commit message and force push the update.

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -823,5 +823,13 @@
     "picture": "hafiz.jpg",
     "website": "https://github.com/ayazhafiz",
     "bio": "Ayaz is an undergraduate student with interests in free software."
+  },
+  "FabianGosebrink": {
+    "name": "Fabian Gosebrink",
+    "picture": "fabiangosebrink.jpg",
+    "twitter": "FabianGosebrink",
+    "website": "https://fabian-gosebrink.com",
+    "bio": "Fabian Gosebrink is Google Developer Expert for Angular and Web Technologies, Microsoft MVP and Web Developer for ASP.NET Core and Angular. As a professional software engineer, consultant and trainer, he advises and supports customers in the implementation of web applications in the front- and backend up to the mobile area.",
```
Comment (kapunahelewong) [inline aio/content/marketing/contributors.json]:
```suggestion
    "bio": "Fabian Gosebrink is a Google Developer Expert for Angular and Web Technologies, Microsoft MVP and Web Developer for ASP.NET Core and Angular. As a professional software engineer, consultant and trainer, he advises and supports customers in the implementation of frontend, backend, and mobile development of web applications."
```
These edits are by no means required, take them only if you agree wit them. 😊

**Comment by kapunahelewong** [comment]
Hi @FabianGosebrink, this commit message should work ok:
```
docs: add GDE profile of Fabian Gosebrink
```

**Comment by Splaktar** [comment]
Yeah, it can be a little bit confusing and I agree that scope was the closest, but it looks good now!

The only thing left to do is to squash your commits, [instructions](https://gist.github.com/kapunahelewong/567c30f7b353fd1c39bbf6d8343e2dca#how-to-squash) (from @kapunahelewong).

File: aio/content/marketing/contributors.json (hunk 1 above)
Comment (FabianGosebrink) [reply aio/content/marketing/contributors.json]: Perfect, thanks!

**Review by Splaktar — APPROVED** [review]
LGTM

**Review by StephenFluin — APPROVED** [review]

**Review by kapunahelewong — APPROVED** [review]

**Comment by atscott** [comment]
@FabianGosebrink could you please rebase this PR and resolve the conflict? Thanks.

File: aio/content/marketing/contributors.json [hunk 2]
```diff
@@ -829,7 +829,14 @@
     "picture": "mike-brocchi.jpg",
     "twitter": "brocco",
     "website": "https://github.com/Brocco",
-    "bio": "Mike (aka Brocco) is a Google Developer Expert in Web Technologies and Angular. He is passionate about teaching and helping others. You can find him online as half of the ngBS live stream and also as a panelist on the Angular Air video broadcast.",
+    "bio": "Mike (aka Brocco) is a Google Developer Expert in Web Technologies and Angular. He is passionate about teaching and helping others. You can find him online as half of the ngBS live stream and also as a panelist on the Angular Air video broadcast."
+  },
```
Comment (Splaktar) [inline aio/content/marketing/contributors.json]: This merge removed `"groups": ["GDE"]` from Mike Brocchi's profile. It needs to be added back 😄

**Review by Splaktar — CHANGES_REQUESTED** [review]

File: aio/content/marketing/contributors.json (hunk 2 above)
Comment (FabianGosebrink) [reply aio/content/marketing/contributors.json]: Yes, I know...sorry!!!

**Comment by FabianGosebrink** [comment]
Sorry for removing the GDE Group from Mike. That was not on purpose. Sorry, was a merge mistake.

**Review by Splaktar — CHANGES_REQUESTED** [review]
Brocchi's groups look good now 👍

File: aio/content/marketing/contributors.json [hunk 3]
```diff
@@ -831,5 +831,21 @@
...
+  "FabianGosebrink": {
+    "name": "Fabian Gosebrink",
+    "picture": "fabiangosebrink.jpg",
+    "twitter": "FabianGosebrink",
+    "website": "https://fabian-gosebrink.com",
+    "bio": "Fabian Gosebrink is a Google Developer Expert for Angular and Web Technologies, Microsoft MVP and Web Developer for ASP.NET Core and Angular. As a professional software engineer, consultant and trainer, he advises and supports customers in the implementation of frontend, backend, and mobile development of web applications.",
+    "groups": ["GDE"]
+  },
+  "FabianGosebrink": {
```
[the remark this answers is not in the document]
Comment (kapunahelewong) [reply aio/content/marketing/contributors.json]: I took the liberty of fixing this for him.

**Comment by kapunahelewong** [comment]
I fixed the duplication, rebased, fixed the conflict, and also fixed a small spacing issue in the entry above.

**Pull request closed without merging**